src/D2L.CodeStyle.Analyzers/D2L.CodeStyle.Analyzers.csproj

```diff
@@ -19,6 +19,7 @@
     <IncludeSymbols>True</IncludeSymbols>
     <Nullable>enable</Nullable>
     <WarningsAsErrors>Nullable</WarningsAsErrors>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>

```

Comment:
Explicitly setting so it doesn't pick up the lms default.  We do want to burn this down and set to true though